video/out/vo.c

```diff
@@ -185,7 +183,7 @@ static bool get_desc(struct m_obj_desc *dst, int index)
         .options = vo->options,
         .options_prefix = vo->options_prefix,
         .global_opts = vo->global_opts,
-        .hidden = vo->encode || !strcmp(vo->name, "opengl-cb"),
+        .hidden = vo->encode || !strcmp(vo->name, "libmpv"),
```

Comment:
some people complained that `mpv --vo=help` won't list opengl-cb/libmpv with cocoa-cb enabled even though it is valid in that case. should we change this?

Reply:
OK, removed that.